agent/agents/supervisor/supervisor_test.go

```diff
@@ -73,16 +73,16 @@ func TestSupervisor(t *testing.T) {
 
 		assertChanges(t, s,
 			&agentpb.StateChangedRequest{AgentId: "noop3", Status: inventorypb.AgentStatus_STARTING},
```

Comment:
Can I ask you why this test was changed?

Reply:
Given that the supervisor confirms whether the process starts successfully, the "RUNNING" state will follow the "STARTING" state before the supervisor initiates another process.